### PR TITLE
GRW-1598 / Feature / Scan for translation keys

### PR DESCRIPTION
**Pull request by alebedev**
## Describe your changes

Add scan-translations script to detect new and unused translations in apps/store

Annotate purchase form configs to correctly detect used keys and namespaces

Replace key concatenation with switch statements

## Justify why they are needed

Better developer experience

Ability to detect unused translation keys to help with append-only dictionaries

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

File: apps/store/public/locales/en/cart.json [hunk 1]
```diff
@@ -8,9 +8,15 @@
   "CHECKOUT_BUTTON": "Checkout",
   "CHECKOUT_PRICE_TOTAL": "Total",
   "DISCOUNT_DURATION_EXPLANATION": "For {{count}} month, then",
+  "DISCOUNT_DURATION_EXPLANATION_one": "",
```
Comment (robinandeer) [inline apps/store/public/locales/en/cart.json:11]: Hmm I wonder if we should avoid "count" variable unless we explicitly want to pluralize translations...?

File: apps/store/src/services/PriceCalculator/data/SE_ACCIDENT.ts [hunk 2]
```diff
@@ -6,16 +6,19 @@ import {
   streetAddressField,
   yourFamilySection,
 } from '@/services/PriceCalculator/formFragments'
+import { setI18nNamespace, tKey } from '@/utils/i18n'
 import { Template } from '../PriceCalculator.types'
 
+setI18nNamespace('purchase-form')
```
Comment (robinandeer) [inline apps/store/src/services/PriceCalculator/data/SE_ACCIDENT.ts:12]: aha does this work outside react? I guess it's safe it we reload the page every time we switch locale, right?

I imagine we move this to the server in the future -- but then I guess server would just give us translated content right away

Comment (alebedev) [reply apps/store/src/services/PriceCalculator/data/SE_ACCIDENT.ts:12]: This function is no-op for scanner.  We need to make sure it's in sync with the call site where we translate the keys

File: apps/store/public/locales/en/cart.json (hunk 1 above)
Comment (alebedev) [reply apps/store/public/locales/en/cart.json:11]: We probably should - created follow-up task to investigate

**Pull request merged**